tests/testthat/helper-tune-package.R

```diff
@@ -26,3 +26,37 @@ helper_objects_tune <- function() {
     svm_mod = svm_mod
   )
 }
+
+# When not interactive, the cli progress bar is occasionally "reset" at random
+# intervals throughout tuning, which means that intermediate counts may appear
+# in snapshots. In an output like:
+#
+# > A | error:   AHHhH
+# There were issues with some computations   A: x2
+# There were issues with some computations   A: x4
+# There were issues with some computations   A: x5
+# There were issues with some computations   A: x5
+#
+# ...we just want what the interactive user would see, e.g.:
+#
+# > A | error:   AHHhH
+# There were issues with some computations   A: x5
+catalog_lines <- function(pattern) {
```

Comment:
Note that since this is in a `tests/testthat/helper-` file, it will be available on `devtools::load_all()`. It also is _not_ available in extratests, as it would be if it was in `R/`, so we have to copy the function in the test file in the linked PR.